_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Multi-option sign-in
+parent: Configuration
+nav_order: 3
+---
+
+# Multi-option sign-in for Dashboards
+
+You can configure the sign-in window for OpenSearch Dashboards to provide either a single option for authenticating users at login or multiple options. Currently, Dashboards supports basic authentication, OpenID Connect, and SAML as choices for multi-option authentication.
+
+## General steps for configuring multi-option sign-in
+
+1. Decide which types of authentication to make available at sign-in.
+1. Configure each authentication type, including an authentication domain for the identification provider (IdP) and the essential settings that give each type sign-in access to OpenSearch Dashboards (see steps for each authentication type in the Configuration section of the security documentation).
+1. Add, enable, and configure multi-option authentication sign-in settings in the `opensearch_dashboards.yml` file.
+
+## Enabling multi-option sign-in
+
+By default, Dashboards provides a single sign-in environment for basic authentication. To enable multiple options for authentication at sign-in, begin by adding `opensearch_security.auth.multiple_auth_enabled` to the `opensearch_dashboards.yml` file and setting it to `true`.
+
+To specify the authentication types for multi-option sign-in, add the `opensearch_security.auth.type` setting to the `opensearch_dashboards.yml` file and enter multiple types as values. When more than one authentication type is added to the setting, the Dashboards sign-in window recognizes multiple types and adjusts to accommodate the sign-in options.
+
+For single sign-in, the authentication type is specified by adding a single type to the setting.
+
+```yml
+opensearch_security.auth.type: "openid"
+```
+
+For multi-option authentication, add values to the setting as an array separated by commas. As a reminder, Dashboards currently supports a combination of basic authentication, OpenID Connect, and SAML as a valid set of values. In the setting, these values are expressed as `"basicauth"`, `"openid"`, and `"saml"`.
```

Comment:
It will be great if we can highlight that `basicauth is mandatory for multiple authentication mode`.